app/ui.js

```diff
@@ -915,6 +915,32 @@ const UI = {
  *   CLIPBOARD
  * ------v------*/
 
+    writeLocalClipboard(text) {
+        if (typeof navigator.clipboard !== "undefined" && typeof navigator.clipboard.readText !== "undefined") {
```

Comment:
Shouldn't that check for navigator.clipboard.writeText?

Reply:
That's correct! It was a mistake of mine. I'll fix this.